packages/perseus/src/widgets/interactive-graphs/graphs/polygon.tsx

```diff
@@ -42,6 +42,7 @@ export const PolygonGraph = (props: Props) => {
     });
 
     const active = hovered || focused || dragging;
+    const [lastMoveTime, setLastMoveTime] = React.useState<number>(0);
```

Comment:
Could we use a ref instead of state here? That would prevent unnecessary re-renders. I believe a ref is appropriate because we will never need to update DOM elements when `lastMoveTime` changes.

Reply:
Yepppppp. Your absolutely right.